LSP.sublime-settings

```diff
@@ -150,30 +186,56 @@
   // - partially override a configuration in the "default_clients", or
   // - create new configurations.
   //
+  // It is **highly recommended** to download the LSP-json helper package. This
+  // is a package that auto-installs a JSON language server that will provide
+  // auto-completions for all keys in this file, and many others as well.
+  // See: https://packagecontrol.io/packages/LSP-json
+  //
   // "clients":
   // {
   //   // Each new server configuration must have the following structure.
   //   "CLIENT_NAME": // some human-friendly name
   //   {
-  //     # Must-have settings (for new configurations):
+  //     # ----------- The following three settings are **required** -----------
+  //
+  //     // Enable or disable this client configuration.
+  //     "enabled": true,
```

Comment:
What went wrong here? this is already in `main`

Reply:
I might have merged incorrectly, in my machine it's showing 7 files changed, here it's 27

Reply:
I forgot to do `fetch` i guess